config/ec/deployment.go

```diff
@@ -0,0 +1,44 @@
+package ec
+
+import (
+	"fmt"
+
+	"github.com/crossplane/terrajet/pkg/config"
+)
+
+// Configure configures individual resources by adding custom
+// ResourceConfigurators.
+func Configure(p *config.Provider) {
```

Comment:
might be worth pulling some of the values below out into `const` in the future 👍🏻

Reply:
Oh ya, good call out 👍